browser/main/Detail/SnippetNoteDetail.js

```diff
@@ -530,6 +530,7 @@ class SnippetNoteDetail extends React.Component {
             fontSize={editorFontSize}
             indentType={config.editor.indentType}
             indentSize={editorIndentSize}
+            lineNumber
```

Comment:
?

Reply:
I thought Snippet notes should always have line numbers. Shouldn't they?
(`lineNumber` is equal to `lineNumber={true}`.)

Reply:
I think a little regarding the specification.

> (lineNumber is equal to lineNumber={true}.)

Understood.

Reply:
So do you want Snippet notes to respect this setting or should they always show the line numbers?